example/src/components/Card.tsx

```diff
@@ -11,6 +11,7 @@ const Card = ({ title, size, ...props }: Props) => {
     <View style={styles.card}>
       <TurboImage
         {...props}
+        cachePolicy="urlCache"
```

Comment:
can we remove this line?

Reply:
@vokhuyetOz TurboImage will cache image in memory by default. I tested it on Android, it did not show APNG image with memory cache.  I don't have much knowledge about APNG. Do you know why?

Reply:
@duguyihou  i resolved and pushed code.  

if ```cachePolicty="memory"``` decode function received ```SourceImageResource```, penfeizhou.android.animation:apng can not create drawable ( it seems to be this library can not handle this data type)

in case ```cachePolicty="urlCache"``` decode function received ```FileImageResource```.

i tested https://github.com/line/apng-drawable can handle good at both cases

Reply:
please take a look again 👯